contracts/cheatsheet.sol

```diff
@@ -428,7 +427,7 @@ contract Test is ITest, AdvTestBase, Storage, ownerNamespace.Owner, Ballot {
 
     /**
      * If you have a function that will be called very often, in order to safe gas..
-     *  - declare functions as external and make use of calldata location for parameters.
+     *  - declare functions as external/public and make use of calldata location for parameters.
```

Comment:
Actually, not only external/public functions can make use of calldata location for parameters. Private/Internal functions can too (assuming another externally called function passes its calldata to it).

maybe it would make sense to write it this way?

```diff
-     *  - declare functions as external and make use of calldata location for parameters.
+     *  - functions that will be called externally can make use of calldata location for parameters.
```

wdyt?

Reply:
It's seems perfect